packages/ui-react/src/pages/ScreenRouting/playlistScreens/PlaylistLiveChannels/PlaylistLiveChannels.tsx

```diff
@@ -134,6 +134,7 @@ const PlaylistLiveChannels: ScreenComponent<Playlist> = ({ data: { feedid, playl
     <ShareButton title={channelMediaItem.title} description={channelMediaItem.description} url={window.location.href} />
   ) : null;
 
+  // @todo
```

Comment:
todo?

Reply:
Ah yes, I wanted to fix the createURL + liveChannelsURL combination, by giving the latter more args/options, but maybe from the routes refactor. I will remove the todo.